oxidation/client/src/components/WorkspaceListItem.vue

```diff
@@ -0,0 +1,224 @@
+        {{ workspace.name }}
+      </ion-label>
+    </div>
+
+    <!-- role user -->
+    <div class="workspace-role">
+      <tag-role
+        :role="workspace.role"
+      />
+    </div>
+
+    <!-- user avatars -->
+    <div class="workspace-users">
+      <avatar-group
+        class="shared-group"
+        :people="workspace.sharedWith"
+        :max-display="2"
+        @click.stop="$emit('shareClick', $event, workspace)"
+      />
+    </div>
+
+    <!-- last update -->
+    <div class="workspace-lastUpdate">
+      <ion-label class="label-last-update cell">
+        {{ timeSince(workspace.lastUpdate, '--', 'short') }}
+      </ion-label>
+    </div>
+
+    <!-- workspace size -->
+    <div class="workspace-size">
+      <ion-label class="label-size cell">
+        {{ fileSize(workspace.size) }}
+      </ion-label>
+    </div>
+
+    <!-- options -->
+    <div class="workspace-options">
+      <ion-button
+        v-if="!isSelected"
+        fill="clear"
+        class="options-button"
+        @click.stop="$emit('menuClick', $event, workspace)"
+      >
+        <ion-icon
+          :icon="ellipsisHorizontal"
+          slot="icon-only"
+          class="options-button__icon"
+        />
+      </ion-button>
+    </div>
+  </ion-item>
+</template>
+
+<script setup lang="ts">
+import {
+  business,
+  ellipsisHorizontal,
+  cloudDone,
+  cloudOffline
+} from 'ionicons/icons';
+import { ref, inject } from 'vue';
+import { IonIcon, IonButton, IonItem, IonLabel } from '@ionic/vue';
+import { FormattersKey } from '@/common/injectionKeys';
+import { MockWorkspace } from '@/common/mocks';
+import AvatarGroup from '@/components/AvatarGroup.vue';
+import TagRole from '@/components/tagRole.vue';
+
+const isSelected = ref(false);
+
+defineProps<{
+  workspace: MockWorkspace
+}>();
+
+defineEmits<{
+  (e: 'click', event: Event, workspace: MockWorkspace): void,
+  (e: 'menuClick', event: Event, workspace: MockWorkspace): void,
+  (e: 'shareClick', event: Event, workspace: MockWorkspace): void
+}>();
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const { timeSince, fileSize } = inject(FormattersKey)!;
+
+</script>
+
+<style lang="scss" scoped>
+
+.workspace-list-item {
+  --background-activated: var(--parsec-color-light-primary-100);
+  border-radius: 4px;
+
+  &:hover {
+    --background-hover: var(--parsec-color-light-primary-30);
+    --background-hover-opacity: 1;
+  }
+
+  &:focus {
+    --background-focused: var(--parsec-color-light-primary-100);
+    --background-focused-opacity: 1;
+  }
+}
+
+.workspace-list-item>[class^="workspace-"] {
+  padding: 0 1rem;
+  display: flex;
+  align-items: center;
+  height: 4rem;
+}
+
+.workspace-name {
+  padding: .75rem 1rem;
+  width: 100%;
+  max-width: 20vw;
+  white-space: nowrap;
+  overflow: hidden;
+
+  &__icons {
+  position: relative;
+  padding: 5px;
+
+    .main-icon {
+      color: var(--parsec-color-light-secondary-text);
+      font-size:1.5rem ;
```

Comment:
- add missing space between ':' and '1.5'
 - remove useless space after 'rem'

Reply:
Done